src/main/webapp/app/exam/participate/exercises/exercise-overview-page/exam-exercise-overview-page.component.html

```diff
@@ -36,6 +36,33 @@ <h2 style="text-align: left; font-weight: normal; margin-bottom: 30px">
             </tr>
         </thead>
         <tbody>
+            @if (quizExam) {
+                <tr>
+                    <td style="max-width: 30px"></td>
+                    <td style="max-width: 30px">
+                        <div [ngbTooltip]="getIconTooltip(ExerciseType.QUIZ) | artemisTranslate">
+                            <fa-icon [icon]="getIcon(ExerciseType.QUIZ)" placement="right auto"></fa-icon>
```

Comment:
Can we somehow avoid these function calls?

Reply:
Yes, it's avoidable. Thanks for catching this. The method call on the template is removed.